### PR TITLE
pin gem versions for older Rubies

### PR DESCRIPTION
**Pull request by mmoll**
also removes pinning for Ruby <2.0

### CODE REVIEW

**Comment by mmoll** [comment]
http://ci.theforeman.org/job/test_proxy_plugin_pull_request/250/ 💚

File: Gemfile [hunk 1]
```diff
@@ -9,12 +9,26 @@ end
 group :test do
   gem 'smart_proxy_dynflow', :path => '.'
   gem 'smart_proxy', :git => "https://github.com/theforeman/smart-proxy", :branch => "develop"
+
+  if RUBY_VERSION < '2.1'
+    gem 'public_suffix', '< 3'
+  else
+    gem 'public_suffix'
```
Comment (adamruzicka) [inline Gemfile]: Why do we suddenly need this?

Comment (mmoll) [reply Gemfile]: webmock -> addressable -> public_suffix

Comment (adamruzicka) [reply Gemfile]: Oh, I see. Thanks for clarifying it

**Comment by adamruzicka** [comment]
Thanks @mmoll !

**Pull request merged**